### ISSUE TITLE
Add ability to set time when starting app

### ISSUE THREAD

**Issue by bbenefield89** [bbenefield89/PythonAlarm#19]
### Expected Behavior
User can set time for alarm when starting Pylarm e.g. `pylarm 2:37pm`

### Proposed Solution (if any)

Add new option in `src/main.py` to handle non-CLI options that would parse the time and set the alarm according to the 12hour format and meridiem given by the user.

**Comment by swastikbanerjee07** [comment]
Hi ! I'm just beginning with open source contributions and I'd like to take this issue up!
Can you please help me how to proceed with it? @bbenefield89

**Comment by bbenefield89** [comment]
Hi, @swastikbanerjee07 thanks for taking an interest in Pylarm.

This particular issue needs updating. As of today, 10/10/2018, a pull-request has been merged into master solving what I believe to be the bulk of this, thanks to @ChatPion.

Do not fret, as this issue is not 100% complete. What you can do is add in the option to set a meridiem, e.g. A.M. or P.M.

Currently, one can use a command such as `python <path to pylarm>/src/main.py 12:00` and Pylarm will decide whether or not this needs to be an A.M. or P.M. and choose the day for the user as well.

What I am suggesting to you is to now add that last bit of functionality so if a user were to supply a meridiem then there would be no need for Pylarm to make that decision for a user.

For example, if a user were to set an alarm for `12:00pm` but it was actually `1:00pm`, the alarm would be set for `12:00pm` the following day. If user also set an alarm for `2:00pm` and it was `1:00pm` then the alarm would be set for one hour from that time.

Now, for some direction I would start playing around with the `/src/main.py` file and split the command line arguments up in a way that Python itself will know what to do with a `pm` or `am` when given a time.

There's already something very similar going on on `line-32` I believe in the `/src/main.py` that is doing a lot of this. Perhaps that can guide you in adding further functionality.

Let me know if you have anymore questions and again thanks for taking an interest.

**Comment by swastikbanerjee07** [comment]
Hi ! 
That's awesome! I'm looking into it!

**Comment by swastikbanerjee07** [comment]
![screenshot from 2018-10-11 00-38-56](https://user-images.githubusercontent.com/31207633/46760038-83c7eb00-ccee-11e8-9452-3a1ec1a9aa09.png)
WhenI'm trying to run the pylarm.py file, it's showing some error.

**Comment by swastikbanerjee07** [comment]
When I'm running the pylarm.py file, it's showing some error
![screenshot from 2018-10-11 00-43-54](https://user-images.githubusercontent.com/31207633/46760123-ca1d4a00-ccee-11e8-9583-d3f665696ddf.png)

**Comment by swastikbanerjee07** [comment]
@bbenefield89  What we can do to solve the issue is, ummn, how about converting the current time and the given time both to 24 hours format(which I can write the code of) and check if the former is greater than the latter. If yes then we can set alarm for the next day, if no we can set alarm for the given time.

**Comment by swastikbanerjee07** [comment]
Hi, so basically I checked the alarm running into my own computer.
As of now, what it does is, it checks whether my given_time is more than or greater than my current_time. If it's more, then it sets up an alarm for an immediate time pertaining to the given time in 12 hours format. For eg, it's 03:04 clock here now(am,i.e midnight), if I set an alarm for 4:00,it sets an alarm for 4am and not for 4:00pm(i.e,16:00) tomorrow.
My first question is, is there a way to change this default? I mean if I want can I set an alarm for 4pm tomorrow also and not for the immediate 4am when it's 3am here?

And I checked if given_time is less than current time in 12 hrs format, then it automatically sets an alarm in 12 hrs format for next day. E.g, I tried setting up an alarm for 2:00 when its 03:00(am) here, and it set up an alarm for 2pm(i.e, 14:00) tomorrow.



So basically, I guess, we have to incorporate this feature of AM/PM here to make it more convenient?

**Comment by swastikbanerjee07** [comment]
@bbenefield89 Are you sure the alarm sets itself for 12pm if it's 1pm here? Or does it set up for 12am if it's 1pm here?:/
Because I think it's setting up for it's immediate 12

**Comment by swastikbanerjee07** [comment]
@bbenefield89 Are you sure the alarm sets itself for 12pm if it's 1pm here? Or does it set up for 12am if it's 1pm here?:/
Because I think it's setting up for it's immediate 12

**Comment by swastikbanerjee07** [comment]
The previous code had various errors, like the set_alarm_manually file has a missing parameter as I have pointed out in an issue.
However, I have debugged all the codes, and the new alarm now works cool and fine ! :)

The alarm now asks for a time in the beginning in 24Hrs format. If you give a time less than your current local time, it automatically tries to set the alarm in the next day ! And if you give a time which is yet to come today,i.e. more than your current time, then it sets the alarm for today.
Before setting the final alarm, it asks for a confirmation. If you press Y , it sets the alarm. If you press N, it then asks you to set the alarm manually on your preferred month,date,year and time. :)

Hope I could make everything clear.

Please review and merge my Pull  Request!
Thanks.
